cmd/notation/sign.go

```diff
@@ -78,7 +92,11 @@ func runSign(command *cobra.Command, cmdOpts *signOpts) error {
 	if err != nil {
 		return err
 	}
-	sigRepo, err := getSignatureRepository(ctx, &cmdOpts.SecureFlagOpts, cmdOpts.reference)
+	var ociImageManifest bool
+	if cmdOpts.imageSpec == imageManifest {
+		ociImageManifest = true
+	}
```

Comment:
nit: code can be simplified
```suggestion
	ociImageManifest := cmdOpts.imageSpec == imageManifest
```